esmvaltool/diag_scripts/hydrology/globwat.py

```diff
@@ -0,0 +1,346 @@
+"""Globwat diagnostic.
+
+Currently, the model reads monthly reference evaporation and then multiplies
+it by kc and then divide it by the number of days in each month and uses it
+as daily actual evaporation in the model. We can change the model code to
+read daily reference evaporation and do the calculation, but we
+did not change the model code for this aim yet.
+"""
+import logging
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+import pandas as pd
+import dask.array as da
+import iris
+
+from esmvalcore.preprocessor import regrid
+from esmvaltool.diag_scripts.hydrology.derive_evspsblpot import debruin_pet
+from esmvaltool.diag_scripts.hydrology.compute_chunks import compute_chunks
+from esmvaltool.diag_scripts.shared import (ProvenanceLogger,
+                                            get_diagnostic_filename,
+                                            group_metadata,
+                                            run_diagnostic)
+
+
+logger = logging.getLogger(Path(__file__).name)
+
+
+def create_provenance_record():
+    """Create a provenance record."""
+    record = {
+        'caption': "Forcings for the GlobWat hydrological model.",
+        'domains': ['global'],
+        'authors': [
+            'abdollahi_banafsheh',
+            'alidoost_sarah',
+        ],
+        'projects': [
+            'ewatercycle',
+        ],
+        'references': [
+            'acknow_project',
```

Comment:
Perhaps you can insert the references here that you also used in the documentation?

Reply:
I added the references.

Reply:
I think the reference is still missing

Reply:
- debruin16ams
    - hoogeveen15hess
    - langbein1949usgs